modules/listing-alerts-that-are-firing.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="listing-alerts-that-are-firing_{context}"]
+= Listing alerts that are firing
+
+Alerts provide notifications that a set of defined conditions are true in an {product-title} cluster. You can review the alerts that are firing in your cluster by using the Alerting UI in the {product-title} web console.
```

Comment:
"Alerts provide notifications _that_ a set" --> "Alerts provide notifications _when_ a set" ?

Reply:
Also, should "Alerting UI" be in bold? Is it a menu option, tab label, or similar?

Reply:
I have resolved the first suggestion.

Regarding the second suggestion, the Alerting UI is a reference to *Monitoring* -> *Alerting* -> *Alerts*, but it is not a menu option, tab label or suchlike itself, as far as I know. The procedure highlights how to navigate to *Monitoring* -> *Alerting* -> *Alerts* and states those menu items in bold. As such, I will leave "Alerting UI" as it is.